plugins/plugin-sass/test/fixtures/sass/App.sass

```diff
@@ -0,0 +1,6 @@
+@use 'base';
+
+.App {
+  text-align: center;
+  background: base.$primary-color;
```

Comment:
No semicolons in `.sass` files (they’re pure indentation, like YAML)

Reply:
Good catch! this actually needed a small fix in the plugin 👍